cmds/netcat/netcat.go

```diff
@@ -18,30 +18,57 @@ import (
 
 const usage = "netcat [go-style network address]"
 
-var netType = flag.String("net", "tcp", "What net type to use, e.g. tcp, unix, etc.")
+var (
+	netType = flag.String("net", "tcp", "What net type to use, e.g. tcp, unix, etc.")
+	listen  = flag.Bool("l", false, "Listen for connections.")
+	verbose = flag.Bool("v", false, "Verbose output.")
+)
 
 func init() {
 	util.Usage(usage)
 }
 
 func main() {
-	flag.Parse()
 	var c net.Conn
 	var err error
-	if len(flag.Args()) < 1 {
+	if flag.Parse(); len(flag.Args()) != 1 {
```

Comment:
I would unembed flag.Parse(); since there is no variable that's just in scope for the if-statement.